src/components/MainComponent.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
instead of adding this to a `MainComponent`, lets create a `Home` component, and add that component into the `pages` directory.

Reply:
10-4 this is where i could benefit from a zoom session just so i understand the flow completely if thats alright